src/core/lang.c

```diff
@@ -8,6 +8,7 @@
 #include "core/string.h"
 #include "translation/translation.h"
 
+#include <bits/stdint-uintn.h>
```

Comment:
Please don't use this non-standard include as it breaks compatibility with many or our platforms.

Reply:
Sure, it was just auto imported for the hardcoded string.